main.tf

```diff
@@ -3,9 +3,9 @@
 // }
 
 // # tfsec 検知用
-// resource "aws_s3_bucket" "bucket" {
-//   acl = "public-read"
-// }
+resource "aws_s3_bucket" "bucket" {
```

Comment:
🚫 **[tfsec]** <[AVD-AWS-0087](https://aquasecurity.github.io/tfsec/v1.1.2/checks/aws/s3/block-public-policy/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>No public access block so not blocking public policies